src/node_crypto.cc

```diff
@@ -5263,7 +5263,7 @@ void PBKDF2(const FunctionCallbackInfo<Value>& args) {
   THROW_AND_RETURN_IF_NOT_BUFFER(args[1], "Salt");
 
   pass = static_cast<char*>(node::Malloc(passlen));
-  if (pass == nullptr) {
+  if (pass == nullptr && passlen > 0) {
     FatalError("node::PBKDF2()", "Out of Memory");
   }
```

Comment:
I thought about that too and that could be useful, but I'm not sure how everyone else would feel about it.